### PR TITLE
DOCS-6154 Saved Views in the Synthetic Monitoring & Continuous Testing Explorer Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Add Saved Views doc page + screenshot in the Explorer section.

### Motivation

DOCS-6154, ready to merge.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.
- [ ] Check that `cache_enabled` is set to `true` in the `pull_config_preview.yaml` file

### CODE REVIEW

File: content/en/continuous_testing/explorer/saved_views.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+## Overview
+
+Saved views allow you to save the state of the [Synthetic Monitoring & Continuous Testing Explorer][2] and enable effective troubleshooting by providing you with quick access to scoped queries, relevant facets, visualization options, and the time range. 
+
+Saved views can keep track of your:
+
+- CI batches and test runs
+- Search queries (such as failed test runs with HTTP error status codes, failing test runs in the CI by their blocking status, test runs that required retries, and test IDs to add to your CI pipeline)
+- Column sort order
```
Comment (HugoPct) [inline content/en/continuous_testing/explorer/saved_views.md:18]: what is this?

File: content/en/continuous_testing/explorer/saved_views.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- Live time range (such as the past hour or the past week)
+- Visualizations (such as a timeseries, toplist, table, or funnel graph)
+- Subset of facets
```
Comment (HugoPct) [inline content/en/continuous_testing/explorer/saved_views.md:21]: facets are filters that are used in the search query, so we can get rid from them here as we talk about search above

**Review by HugoPct — APPROVED** [review]
left 2 comments otherwise good to go

File: content/en/continuous_testing/explorer/saved_views.md (hunk 2 above)
Comment (alai97) [reply content/en/continuous_testing/explorer/saved_views.md:21]:
```suggestion
```

File: content/en/continuous_testing/explorer/saved_views.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+kind: documentation
+further_reading:
+- link: "/continuous_testing/explorer/search_syntax/"
+  tag: "Documentation"
+  text: "Learn how to create a search query"
+---
+
+## Overview
+
+Saved views allow you to save the state of the [Synthetic Monitoring & Continuous Testing Explorer][2] and enable effective troubleshooting by providing you with quick access to scoped queries, relevant facets, visualization options, and the time range.
```
Comment (alai97) [inline content/en/continuous_testing/explorer/saved_views.md:12]:
```suggestion
Saved views allow you to save the state of the [Synthetic Monitoring & Continuous Testing Explorer][2] and enable effective troubleshooting by providing you with access to scoped queries, relevant facets, visualization options, and the time range. 
```

File: content/en/continuous_testing/explorer/saved_views.md (hunk 1 above)
Comment (HugoPct) [reply content/en/continuous_testing/explorer/saved_views.md:18]: We can remove it and add list in the list of visualisations

File: content/en/continuous_testing/explorer/saved_views.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Saved views allow you to save the state of the [Synthetic Monitoring & Continuous Testing Explorer][2] and enable effective troubleshooting by providing you with quick access to scoped queries, relevant facets, visualization options, and the time range. 
+
+Saved views can keep track of your:
+
+- CI batches and test runs
+- Search queries (such as failed test runs with HTTP error status codes, failing test runs in the CI by their blocking status, test runs that required retries, and test IDs to add to your CI pipeline)
+- Column sort order
+- Live time range (such as the past hour or the past week)
+- Visualizations (such as a timeseries, toplist, table, or funnel graph)
```
[the remark this answers is not in the document]
Comment (HugoPct) [reply content/en/continuous_testing/explorer/saved_views.md]: no funnel graph

File: content/en/continuous_testing/explorer/saved_views.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+## Overview
+
+Saved views allow you to save the state of the [Synthetic Monitoring & Continuous Testing Explorer][2] and enable effective troubleshooting by providing you with access to scoped queries, relevant facets, visualization options, and the time range. 
+
+Saved views can keep track of your:
+
+- CI batches and test runs
+- Search queries (such as failed test runs with HTTP error status codes, failing test runs in the CI by their blocking status, test runs that required retries, and test IDs to add to your CI pipeline)
+- Column sort order
```
Comment (HugoPct) [inline content/en/continuous_testing/explorer/saved_views.md:18]:
```suggestion
```

**Review by HugoPct — APPROVED** [review]

**Review by rtrieu — APPROVED** [review]
Looks great!

**Pull request merged**